src/Apps/Artist/Routes/WorksForSale/Components/__tests__/ArtistArtworkFilter.jest.tsx

```diff
@@ -117,7 +117,7 @@ describe("ArtistArtworkFilter", () => {
       const option = screen.getByRole("option", { name: "Default" })
 
       expect(option).toBeInTheDocument()
-      expect(option).toHaveValue("-default_trending_score")
+      expect(option).toHaveValue("-decayed_merch")
```

Comment:
**suggestion**: i think you can remove [all this case](https://github.com/artsy/force/blob/2dbc28db82bb187a92a54190633c3d5a2f256649/src/Apps/Artist/Routes/WorksForSale/Components/__tests__/ArtistArtworkFilter.jest.tsx#L78)

Reply:
🙈 you're definitely right

Reply:
🙈 you're definitely right